planning/motion_velocity_planner/autoware_motion_velocity_obstacle_cruise_module/config/obstacle_cruise.param.yaml

```diff
@@ -95,6 +95,17 @@
             bicycle: false
             pedestrian: false
 
+          # If an object does not have this label, the ego vehicle does not assume that the yielded obstacle will cut in due to the object.
+          ahead_stopped:
```

Comment:
ahead sounds like the obstalce is ahead of the ego, but on the ego's trajectory.
How about `side_stopped` instead?

Reply:
Thank you! It sound good for me. I will update parameters soon.

Reply:
I updated the parameter in 90f6046